src/main/resources/db/migration/V1.21__Create_Legacy_Beacons_Table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS legacy_beacon_records (
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS legacy_beacon (
```
wondering if that would match up better with the other `beacon` table 🤔